spec/index.bs

```diff
@@ -12176,6 +12176,9 @@ These operations are encoded within {{GPUComputePassEncoder}} as:
 
 Issue: describe the computing algorithm
 
+If [[WGSL#shader-execution-end|shader execution does not end]],
+then a {{GPUError}} may be issued, or the [=device=] may become [=lose the device|lost=].
```

Comment:
If we allow a GPUError, we should be specific about which one (maybe #3123).

However if we allow that we also need to be very precise about what does and doesn't happen. Does the next call in the pass happen? Is the whole pass cancelled? Is the whole command buffer cancelled? In any of those cases, do the stores at the end of the render pass happen?

I suspect there are no strong guarantees from underlying APIs in cases like this and we should simply allow only device lost for now.

Reply:
Right.  I was guessing.   From what I know, it's probably only context lost.  I'll remove the GPUError and see if that's more palatable.